### ISSUE TITLE
Pino reports my own tweet, not libnotify

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#151]
```
If a post a new tweet the icon will change to notify me about my own new
tweet, libnotify doesnt notify me though.

Im using Arch and Pino 0.2.3 from the mercurial repo.
```

Original issue reported on code.google.com by `58desi...@gmail.com` on 27 Mar 2010 at 9:45

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 27 Mar 2010 at 9:57
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 78297d7208 revision.
```

Original comment by `tro...@gmail.com` on 3 Apr 2010 at 11:09
- Changed state: **Fixed**
